### PR TITLE
{bio}[foss/2019b] HISAT2 v2.2.1

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/h/HISAT2/HISAT2-2.2.1-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+local_executables = ['hisat2', 'hisat2-align-l', 'hisat2-align-s', 'hisat2-build', 'hisat2-build-l', 'hisat2-build-s',
+                     'hisat2-inspect', 'hisat2-inspect-s', 'hisat2-inspect-l', 'hisat2-repeat', 'extract_exons.py',
+                     'extract_splice_sites.py', 'hisat2_extract_exons.py', 'hisat2_extract_snps_haplotypes_UCSC.py',
+                     'hisat2_extract_snps_haplotypes_VCF.py', 'hisat2_extract_splice_sites.py',
+                     'hisat2_read_statistics.py', 'hisat2_simulate_reads.py']
+files_to_copy = [
+    (local_executables, 'bin'),
+    ('scripts'),
+    ('example'),
+]
```
Comment (smoors) [inline easybuild/easyconfigs/h/HISAT2/HISAT2-2.2.1-foss-2019b.eb:47]: @verdurin the parentheses are not needed if you copy to the root installdir, so you can just do:
```suggestion
files_to_copy = [(local_executables, 'bin'), 'scripts', 'example']
```

**Review by smoors — DISMISSED** [review]

File: easybuild/easyconfigs/h/HISAT2/HISAT2-2.2.1-foss-2019b.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/h/HISAT2/HISAT2-2.2.1-foss-2019b.eb]: @smoors done.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node367.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c4b6928ff7eddc05efd2cf6e4366d74e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/edefb370e7a66c65e91e62f4a0435811 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @verdurin !

**Pull request merged**